src/index.js

```diff
@@ -41,6 +41,15 @@ async function isWithinRange(forwardedFor, fastlyPublicIps, whitelistedIps = '')
     .some((myranges) => (range.isRange ? range.inRange(ip, myranges) : range === ip)));
 }
 
+async function getEmbedKind(domainList) {
+  const cpy = domainList;
+  // remove first level domain
+  cpy.pop();
```

Comment:
what's with `www.youtube.co.uk`  ?

Reply:
Good point! Lots of countries have `co.*`, so we should check for `co` at the 2nd level.